testing/munin/munin-node.initd

```diff
@@ -0,0 +1,36 @@
+#!/sbin/openrc-run
+
+# munin-node init.d file for alpine linux.
+
+name=munin-node
+
+pidfile=/var/run/munin/${name}.pid
+
+command=/usr/sbin/munin-node
+command_args="${EXTRA_OPTS}"
+
+depend() {
+  need net
+  after firewall
+}
+
+start() {
+  if [ ! -d /var/run/munin ]; then
+    mkdir -p /var/run/munin
+    chown munin:munin /var/run/munin
+  fi
```

Comment:
1. This should be in `start_pre()`, not `start()` ([man openrc-run](http://manpages.org/openrc-run/8)).
2. openrc-run provides handy function for this: `checkpath -d -o munin:munin -m755 /run/munin` ([man openrc-run](http://manpages.org/openrc-run/8)).